src/data/README.md

```diff
@@ -1,7 +1,23 @@
-# Data
+# OpenThaiGPT Dataset Pipeline Overview
 
 Folder `openthaigpt_pretraining_data` contains the preprocess function, EDA notebook, and examples for each dataset
 
 Folder `notebook` contains the EDA or other experiment notebook.
 
 Folder `scripts` contains the main program which will call function in folder `openthaigpt_pretraining_data`
+
+## Description
+
+The OpenThaiGPT Overview dataset pipeline is meticulously designed to fully preprocess raw data, extract essential information, and prepare the dataset in a trainable format. Here's an extended breakdown of its key components:
+
+## Workflow
+
+1. Obtain Raw Data in various formats such as CSV, PDF, etc., and convert it into plain text format.
+2. Transform plain text data from each raw data source into the JSONL format.
```

Comment:
We also have scripts for this krub

not all script is already in git for now, but the one you should include in this readme are: https://github.com/OpenThaiGPT/openthaigpt-pretraining/tree/main/src/data/scripts/pantip_2G
https://github.com/OpenThaiGPT/openthaigpt-pretraining/tree/main/src/data/scripts/pantip_3G
https://github.com/OpenThaiGPT/openthaigpt-pretraining/tree/main/src/data/scripts/internet